### DISCUSSION TITLE
Text & Options Field in one

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#8433] [Ideas]
Hey,

I just thought about a text/options field where you can either pick a option or write something.
Is that a possibility right now?

Or when creating a new row check a value against values from existing rows*

Sincerely,
Kevin

**Comment by melohagan** [comment]
This is a great idea. It's not possible at the moment but there is an open issue: https://github.com/Budibase/budibase/issues/6889

It could also potentially be built as a custom component depending on your coding appetite: https://docs.budibase.com/docs/custom-component#example-custom-form-field-component
